src/test/java/FormatterTest.java

```diff
@@ -12,6 +12,8 @@ class FormatterTest {
     String singleTypeExample =
             "Joe1      Smith     Developer 07500010012009\n" +
             "Joe3      Smith     Developer ";
+    String badLocalDateExample =
+            "Joe1      Smith     Developer 07500015012009";
```

Comment:
15 01 2009 is an invalid date because the 15th month does not exist.

Reply:
Ah, it is because of US date format. You could use your own set of formatters though.

Reply:
https://github.com/g0ddest/fixedlength/blob/master/src/main/java/name/velikodniy/vitaliy/fixedlength/formatters/LocalDateFormatter.java
Library has strictly defined date format as yyyyMMdd. We also could add additional formatting field to annotation that specifies format.
@spardue , I found a few solutions, does them work for you?

Reply:
@g0ddest I think the issue is that sometimes the fields given can be in an invalid format and which stops the entire process because the exception is thrown. I could probably just not use LocalDate and use a String and have custom checks for dates.

Reply:
@spardue, sure, you could use this String in your payload, but I see no problem having LocalDate if we are sure that there is no problems in our data. I believe that haveing error on broken files is ok. If file is broken so you could choose behaviour on your business requirements using own formatters, see examples how to implement one in code. In the test case there is no problem in that and date valid by ISO format, that is specified in LocalDate format.